vaadin-dashboard-flow-parent/vaadin-dashboard-testbench/src/main/java/com/vaadin/flow/component/dashboard/testbench/DashboardWidgetElement.java

```diff
@@ -25,4 +25,21 @@ public class DashboardWidgetElement extends TestBenchElement {
     public String getTitle() {
         return getPropertyString("widgetTitle");
     }
+
+    /**
+     * Returns the colspan of the widget.
+     *
+     * @return the {@code --vaadin-dashboard-item-colspan} style from the
+     *         wrapper of the web component
```

Comment:
The new implementation isn't getting the value from the wrapper.

Reply:
Updated the wording.